src/playActivity/playActivityDB.h

```diff
@@ -442,7 +442,7 @@ void play_activity_db_V3_upgrade(void)
           
             if (db_handles[nDB] != NULL) {
 
-                sql  = sqlite3_mprintf("SELECT * FROM %q_roms WHERE path LIKE '%%%q%%' LIMIT 1;", db_handles_type[nDB], rom_list[i].name);                 
+                sql  = sqlite3_mprintf("SELECT * FROM %q_roms WHERE path LIKE '%%%q' LIMIT 1;", db_handles_type[nDB], rom_list[i].name);                 
```

Comment:
Don't you need the wildcard here, since `rom_list[i].name` doesn't contain the file extension?